.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: go
 go:
-  - 1.1
-  - 1.2
-  - 1.3
-  - tip
\ No newline at end of file
+  - 1.1.x
+  - 1.2.x
+  - 1.3.x
+  - 1.4.x
+  - 1.5.x
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
+  - 1.10.x
+  - 1.11.x
```

Comment:
Please remove all except this following.
``` 
 - 1.11.x
 - tip
```

Reply:
@mattn Updated.